pkg/apis/app/v2/kieapp_types.go

```diff
@@ -894,6 +895,8 @@ type CommonConfig struct {
 	DisableSsl bool `json:"disableSsl,omitempty"`
 	// Startup strategy for Console and Kieserver
 	StartupStrategy *StartupStrategy `json:"startupStrategy,omitempty"`
+	// Is the name of the secret containing admin's credentials, the keys are username and password, if the username and password are provided the secretAdminCredentials will be skipped
```

Comment:
It is confusing when there used same names for keys and properties. Also properties are named differently so in that case it would be much better and cleaner to use the property names from KieApp.
```suggestion
	// Is the name of the secret containing admin's credentials, the keys are username and password, if the adminUser and adminPassword are provided the secretAdminCredentials will be skipped
```